engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.controller.js

```diff
@@ -122,13 +122,13 @@ angular.module('Bastion.capsule-content').controller('CapsuleContentController',
 
         $scope.isTaskInProgress = isTaskInProgress;
 
-        $scope.syncCapsule = function () {
+        $scope.syncCapsule = function (skipMetadataCheck = false) {
```

Comment:
@waldenraines the linter doesn't seem to like this default argument, is there another way I should be writing this?
```
Running "eslint:target" (eslint) task

app/assets/javascripts/bastion_katello/capsule-content/capsule-content.controller.js
  125:58  error  Unexpected token =

✖ 1 problem (1 error, 0 warnings)

```